app.py

```diff
@@ -184,24 +184,14 @@ def logout_():
     @authenticated(ADMINS_GROUP)
     @csrf_needed
     def push_log():
-        if 'level' not in request.json or 'message' not in request.json:
-            raise ValueError('Request body missing one or more mandatory fields ["message", "level"]')
+        if type(request.json) != list:
```

Comment:
question: why are we not accepting a single entry anymore?

Reply:
Switching to a buffering logger all log entries would go through the buffering process so the previous logic would not be needed anymore. Of course we can easily reintroduce it if we need it again